test/karma.conf.js

```diff
@@ -51,21 +52,34 @@ module.exports = function(config) {
       'src/components/DistortableImage.Keymapper.js',
       'test/SpecHelper.js',
       'test/src/*Spec.js',
-      'test/src/**/*Spec.js',
+      'test/src/**/*Spec.js'
     ],
 
     // so that karma can serve examples/example.png
     proxies: {
-      '/examples/': '/base/examples/',
+      '/examples/': '/base/examples/'
     },
 
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
     reporters: ['mocha', 'coverage'],
 
+    babelPreprocessor: {
+      options: {
+        presets: ['@babel/preset-env'],
+        sourceMap: 'inline'
+      },
+      filename: function(file) {
+        return file.originalPath.replace(/\.js$/, '.es5.js');
+      },
+      sourceFileName: function(file) {
+        return file.originalPath;
+      }
+    },
+    
     preprocessors: {
-      '../src/**/*.js': 'coverage',
+      'src/**/*.js': ['babel']
```

Comment:
Any idea if the `coverage` task was even doing anything in the 1st place? When I run `$ grunt build` on `main` for ex., it seems it doesn't..? Should we remove it from our Gruntfile completely if that is the case? @jywarren @rexagod

Reply:
As far as I know, it's not doing anything. coverage.txt is always empty.

Reply:
Ok that can be a separate PR because that doesn't have to do with your PR. I'll open an issue for it now. Did you want to open that PR or should I?

Reply:
If you want to try to fix it, feel free to do so. If not, I'll fix once I wake up in the morning (it's 3AM here :)

Reply:
get some rest!!! I opened the issue but I would hold on adding a PR for it until this is merged. But I will let u open it!

Reply:
I've fixed it.